<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpdateCommodity</class>
 <widget class="QDialog" name="UpdateCommodity">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>399</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>更新商品信息</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>521</width>
     <height>381</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QComboBox" name="choice">
        <item>
         <property name="text">
          <string>请选择更新方式</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>编号</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>名称</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>分类</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>用途</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>库存量</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>仓库所在地</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>仓库编号</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="oldInformation">
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>请输入更新前的信息</string>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="updateInformation">
        <property name="placeholderText">
         <string>请输入更新后的信息</string>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="updateButton">
        <property name="text">
         <string>更新</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableView" name="tableViewInformation"/>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
